### ISSUE TITLE
Add support for phrase searching

### ISSUE THREAD

**Issue by thetorpedodog** [bluesky-social/social-app#1143]
**Problem:** Sometimes I am trying to find a post with a phrase that I remember seeing (often one of my own). However, when searching for a phrase, I find every post which contains all the words in the phrase.

**Suggested solution:** I would like to be able to use `"quotation marks"` to search for an exact phrase. For instance, the above query would match a post which says “I hate when I see straight quotation marks in printed material ”, but not match a post which says “On your marks, get set, quotation”.

**Alternatives and context:** I don’t have any alternatives in mind; this is a pretty standard search feature. However, this feature would be particularly useful in conjunction with other search operators, like `-` to exclude matches, or things like `from:`.

**Comment by pfrazee** [comment]
cc @whyrusleeping

**Comment by waxpancake** [comment]
Reviving this two-year-old issue to note that phrase searching DOES work on Bluesky, but the default search form on the iOS app uses smart quotes (aka “curly quotes”), which are ignored. If you push/hold the double-quote key to use "straight quotes" instead, the exact phrase search will work as intended.

**Comment by surfdude29** [comment]
Yes phrase searching does indeed work now, and the smart quotes issue will be fixed by #8865.

**Comment by surfdude29** [comment]
Given that #8865 has been merged, I think this can probably be closed now?
